### PR TITLE
[SCS] update to v3.2.0

### PR DESCRIPTION
**Pull request by kalmarek**
Before we merge this let me check that the binaries actually work with SCS.jl

### CODE REVIEW

**Comment by giordano** [comment]
Marking as draft until it's ready

File: S/SCS_GPU/build_tarballs.jl [hunk 1]
```diff
@@ -2,18 +2,18 @@ using Pkg
...
+    GitSource("https://github.com/cvxgrp/scs.git", "ac6840a3b3264950e6c300264cbf3937e0bcc6c5")
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir/scs*
-flags="DLONG=0 USE_OPENMP=0 BLAS64=1 BLASSUFFIX=_64_"
-blasldflags="-L${libdir} -lopenblas64_"
+flags="DLONG=0 USE_OPENMP=0"
+blasldflags="-L${prefix}/lib -lopenblas"
```
Comment (giordano) [inline S/SCS_GPU/build_tarballs.jl:16]: Why changing `${libdir}` to `${prefix}/lib`?
```suggestion
blasldflags="-L${libdir} -lopenblas"
```

Comment (kalmarek) [reply S/SCS_GPU/build_tarballs.jl:16]: hmm, I copied this from SCS `build_tarballs.jl`. I guess both should point to `${libdir}`?

Comment (giordano) [reply S/SCS_GPU/build_tarballs.jl:16]: > I guess both should point to `${libdir}`?

Not always.   It's preferable to point to the most specific and semantically relevant environment variable.  Also, it was `${libdir}` before, I see no point in changing it

Comment (kalmarek) [reply S/SCS_GPU/build_tarballs.jl]: ok, so what is the reason to have `-L${prefix}/lib` in SCS `build_tarballs.jl`? i.e. what is the canonical dir where all deps are installed?

btw. thanks for taking time to review this ;)

Comment (giordano) [reply S/SCS_GPU/build_tarballs.jl]: > ok, so what is the reason to have -L${prefix}/lib in SCS build_tarballs.jl?

None, that's wrong as far as I'm concerned.  I guess I overlooked it before :slightly_smiling_face: 

>  i.e. what is the canonical dir where all deps are installed?

`${libdir}`

Comment (kalmarek) [reply S/SCS_GPU/build_tarballs.jl]: verywell, I'll change it there the next time. thanks!

**Pull request merged**